vllm/model_executor/models/llama4_eagle.py

```diff
@@ -0,0 +1,199 @@
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: Copyright contributors to the vLLM project
```

Comment:
Can you also add the copyright from Meta side?

Reply:
Absolutely, thanks for the suggestion

Reply:
Example to follow: https://github.com/vllm-project/vllm/blob/main/vllm/model_executor/models/llama4.py